test/all/index.js

```diff
@@ -159,6 +164,31 @@ test('createFile', function(t) {
   });
 });
 
+test('upload', function(t) {
+  var encrypted =false;
+  var stored = false;
+  const rs = new Readable();
+  rs._read = function() {};
+  rs.push(fileC2);
+  rs.push(null);
+  const uploadStream = storj.upload(bucketId, file2);
+  rs.pipe(uploadStream);
+  t.equal(uploadStream.readable, true, 'returned stream is readable');
+  t.equal(uploadStream.writable, true, 'returned stream is writable');
+  uploadStream.on('encrypted', function cb() {
+    encrypted = true;
+  });
+  uploadStream.on('stored', function cb() {
+    stored = true;
+  });
+  uploadStream.on('error', t.fail)
+  uploadStream.on('done', function() {
+    t.equal(stored, true, 'store finished getting encrypted file');
+    t.equal(encrypted, true, 'stream finished encrypting the file');
+    t.end();
+  });
+});
```

Comment:
It doesn't look like we are verifying the uploaded file. Though moving to the proposal above would accomplish that.